### ISSUE TITLE
Weapon: multitool

### ISSUE THREAD

**Issue by avaer** [webaverse/app#1343]
The idea is to have a multitool that can switch modes at the top, by having 8 directional "petals" that can be shown/hidden.

![オリジナル-SFな武器集2-Spark621のイラスト-pixiv-_1_](https://user-images.githubusercontent.com/6926057/124646161-4de5d000-de62-11eb-8b56-98030b5f334a.png)

https://user-images.githubusercontent.com/6926057/124647497-095b3400-de64-11eb-9074-0e1cba22221a.mp4

![Scythe](https://cdn.discordapp.com/attachments/812364944468607046/860222407582023680/unknown.png)

**Comment by avaer** [comment]
https://www.pinterest.ca/pin/685321268300152650/

**Comment by avaer** [comment]
![Shichimen-Warrior-Art-Sekiro_-Shadows-Die-Twice-Art-Gallery](https://user-images.githubusercontent.com/6926057/124648782-aa96ba00-de65-11eb-8221-01d0c99fec0f.png)

**Comment by avaer** [comment]
![Inspiration_-on-We-Heart-It](https://user-images.githubusercontent.com/6926057/124648961-e2056680-de65-11eb-94ab-c4f621e29fdf.png)

**Comment by Vianvolaeus** [comment]
This is still ongoing, but it might be worth waiting on the concept for the character that actually uses (created?) the weapon before iterating on the style massively - the rigging aspect is time consuming to redo.

Do we have a pet / creature that the blades could be made from? 
I'm also considering **basilisk plumes** for theming. Correct shape, fits the adventure:
![image](https://user-images.githubusercontent.com/80061436/124795264-c1add880-df47-11eb-8bc9-64839b7f2357.png)

**Comment by avaer** [comment]
Apparently this character is not actually related to the basilisk (the existence of which is doubtful itself) and this weapon is self created.

However that is worth rewriting if it strengthens themes.

It seems it would be cool for the self creation of the multitool to be a lie — and all of the main weapons are actually basilisk tied.

Here are some suggestions for putting basilisk parts in all of the weapons:

heart of the sword (this has already been suggested)
bones of the scissor
jaw of the scissor
spine of the leash
fang of the leash
crest of the multitool

**Comment by avaer** [comment]
I don’t think it needs to be too tied to the character. Just the fact that it is a multitool would be a reflection of personality.

I’m down for refactoring the lore to fit the above idea if it works visually.

**Comment by Vianvolaeus** [comment]
> It seems it would be cool for the self creation of the multitool to be a lie — and all of the main weapons are actually basilisk tied.

This would serve as a good introductory piece for the animal tamer character, presuming the MC meets them in 1-4 order - they could recognize that everyone else has basilisk parts in their weapons, etc

I think leaning in this direction might be a good idea for the weapons. 
It gives them all a common theme in the Basilisk, which is the common reasoning for our characters to be together in the first place, but all with their unique flair / personality... like the party itself. 
 
 I think it works well for a thematic and narrative standpoint.

**Comment by avaer** [comment]
The narrative seems to like a more gradual reveal. The trailer has everyone meeting immediately, but without recognizing the significance.

In some variations there is a later event in which other characters (besides MC + 2) acquire their weapons.

I'm not 100% on this though, because it seems to detract from the "normality" of 3 and really doesn't jive with anything 4 cares about.

**Comment by GaladWarder** [comment]
I would suggest maybe having a color, palette, or shape language that is tied specifically to the Basilisk design, which can then be used in the weapons.

 It'll tie them together thematically but won't be as obvious as just literal basilisk bones. It'll also be something that 'clicks' upon the first reveal of the Basilisk to the audience, seeing the similar shapes and colors that are shared between it and the weapons without needing any explanation up-front.
 
 I'm planning on tackling the open weapon designs later this week, so let me know any thoughts on this - otherwise I'll just do a range from literal bones to subtle shape.
 
 Also, if these weapons are needing to be tied to the Lisk for narrative reasons, then we should probably nail down at least a preliminary design for the Lisk first, since the weapons are derivative from that

**Comment by avaer** [comment]
I think multitool and the leash don't need to be tied to Lisk, but sword and scissors do.

It would be fine if they all evoked it in some way, but I wouldn't let that get in the way of a workable multitool.

**Comment by avaer** [comment]
What we should do though is follow color coding for the characters. Engineer is red, so weapon should be reddish.
